userguide/platform/includes/killbill-configuration-properties-table.adoc

```diff
@@ -510,7 +510,7 @@ KB_org_killbill_billing_server_event_post_type_skip   |List of event types to be
 KB_org_killbill_billing_server_event_dispatch_type_skip   |List of event types to be skipped (not dispatched internally)   |-   |Per-Tenant/Config File/Environment Variable
 |org.killbill.billing.server.event.bulk.subscription.aggregate/
 
-KB_org_killbill_billing_server_event_bulk_subscription_aggregate   |Coming Soon  |false   |Per-Tenant/Config File/Environment Variable
+KB_org_killbill_billing_server_event_bulk_subscription_aggregate   |Aggregate the events for bulk subscription  |false   |Per-Tenant/Config File/Environment Variable
```

Comment:
Maybe create a PR to also update it [here](https://github.com/killbill/killbill/blob/33aa3e9a65bf40de10edee7429341370488d4fb5/util/src/main/java/org/killbill/billing/util/config/definition/EventConfig.java#L53)?

Reply:
Hi Reshma, Here it is : https://github.com/killbill/killbill/pull/1936